src/helpers/date.js

```diff
@@ -1,16 +1,27 @@
-import dayjs from "dayjs";
+import dayjs, { isDayjs } from "dayjs";
 import LocalizedFormat from "dayjs/plugin/localizedFormat";
 import customParseFormat from "dayjs/plugin/customParseFormat";
 import quarterOfYear from "dayjs/plugin/quarterOfYear";
+import utc from "dayjs/plugin/utc";
+import timezone from "dayjs/plugin/timezone";
 
 dayjs.extend(customParseFormat);
 dayjs.extend(LocalizedFormat);
 dayjs.extend(quarterOfYear);
+dayjs.extend(utc);
+dayjs.extend(timezone);
+
+const DayTimeStart = "T00:00:00";
+const DayTimeEnd = "T23:59:59";
 
 export const formatDate = (date, format = "YYYY-MM-DD") => {
     return dayjs(date).format(format);
 };
 
+export const formatDateWithTimezone = (date, format = "YYYY-MM-DD") => {
+    return dayjs.tz(date).format(format);
+};
```

Comment:
1. You should just use `formatDate` instead of `.format()`
2. `formatDate` should use `now()`
3. Format should not be a string, it should be something from `DateFormat` so `format = DateFormat.UTC` or whatever

Reply:
the reason i didnt use formatDate is it uses dayjs(), and formatDate from UI kit is being used in seller app a lot. eg: **apps/seller/src/components/Dashboard/Filters/DashboardDatePicker.tsx**

Reply:
these are the components which we are not using timezone currently